lib/spreewald_support/mail_to_plaintext_converter.rb

```diff
@@ -0,0 +1,40 @@
+require 'nokogiri'
+
+class MailToPlaintextConverter
```

Comment:
Please wrap this class in a `Spreewald` module, to avoid polluting the global namespace (which Spreewald already does left and right, I know).